src/ds5/advanced_mode/json_loader.hpp

```diff
@@ -403,6 +405,7 @@ namespace librealsense
         insert_string_control_to_map(map, p.depth_auto_exposure.vals[0].was_set, "controls-autoexposure-auto", p.depth_auto_exposure, &auto_exposure_control::auto_exposure, auto_control_values);
 
         insert_control_to_map(map, p.depth_gain.vals[0].was_set, "controls-depth-gain", p.depth_gain, &gain_control::gain);
+        insert_string_control_to_map(map, p.depth_auto_white_balance.vals[0].was_set, "controls-depth-white-balance-auto", p.depth_auto_white_balance, &auto_white_balance_control::auto_white_balance, auto_control_values);

```

Comment:
shouldnt it be "controls-depth-**auto**-white-balance-~~auto~~"?

Reply:
The controls naming convention was followed by old control names template such as 'controls-autoexposure-auto'/'controls-autoexposure-manual'. Changing the controls name will be a little bit painful because we want to save the backward compatibility of advanced-mode JSON files.